addons/mail/static/src/xml/followers.xml

```diff
@@ -38,9 +38,9 @@
         <div role="menuitem" class="o_partner">
             <a class="dropdown-item o_mail_redirect"
                href="#"
-               t-att-title="record.name"
+               t-att-title="record.name or record.display_name"
                t-att-data-oe-model="record.res_model"
-               t-att-data-oe-id="record.res_id"><t t-esc="record.name"/></a>
+               t-att-data-oe-id="record.res_id"><t t-esc="record.name or record.display_name"/></a>
```

Comment:
Actually I was wrong about v14, we did use `nameOrDisplayName` in other places but not in follower, so I guess forward-port would still be in order.

Reply:
Yes, since this issue can also be seen in Master.